charts/karpenter/values.yaml

```diff
@@ -94,11 +94,13 @@ extraVolumes: []
 controller:
   image:
     # -- Repository path to the controller image.
-    repository: public.ecr.aws/karpenter/controller
+    repository: ksnap.azurecr.io/karpenter/snapshot/controller
     # -- Tag of the controller image.
-    tag: v0.32.1
+    tag: 41f6dcabe655c7fec84f4467e95bbd75285ae082
```

Comment:
Assuming this will get reverted for PR completion

Reply:
Reverting, though should be updated to proper release version (with next release)